.vscode/tasks.json

```diff
@@ -9,13 +9,13 @@
 			"isBackground": true,
 			"dependsOn": "pip install (functions)",
 			"options": {
-				"cwd": "${workspaceFolder}/code/backend/batch"
+				"cwd": "${workspaceFolder}"
 			}
 		},
 		{
 			"label": "pip install (functions)",
```

Comment:
Can we condense these into one task, since they're all just running `pip install .`? Also, does `pip install .` still work when you're not in the root of the repo?

Reply:
Combined it to one and `pip install .` only works when in the root of the repo. Basically it looks for the `pyproject.toml` file.